### ISSUE TITLE
parameter of list type read as box.box_list.BoxList

### ISSUE THREAD

**Issue by JLouSRM** [PrefectHQ/prefect#2377]
## Description
When starting a flow run using Cloud UI, we found a list parameter was read as **box.box_list.BoxList**. However, if we hit "quick run" button, then it will be read as a **list**. It is not a big issue to cause any trouble for now, but consistent behavior is expected to generate stable and consistent results. 

## Expected Behavior
A parameter of list type should be always read as list.

## Reproduction
@task
def get_type(any_list):
    print(type(any_list))
    return type(any_list)

with Flow("test") as flow:
    a_list = Parameter("a_list", default=["a","b","c"])
    results = get_type(a_list)

Deploy to cloud and then start the flow by two methods:

1. hit "quick run ", and found "list" in the logging.
2. hit "run" with the default parameter, and then found "<class 'box.box_list.BoxList'> " in the logging.

## Environment
We use a long-running Dask cluster on k8s via AWS EKS.

**Comment by lauralorenz** [comment]
Hi @JLouSRM!

I wasn't able to reproduce this with either Prefect Core's server or Prefect Cloud (full disclosure, using a local agent). I used your example flow except I changed the print statement to log at `CRITICAL` level, and this screenshot then conveniently shows those logs elevated into the 'warnings' tile with the output type, for two flow runs.

"outrageous-anaconda" is when I used the "quick run" button, and "from regular run" is from when I used the normal "Run" tab:

<img width="907" alt="Screen Shot 2020-04-29 at 3 42 29 PM" src="https://user-images.githubusercontent.com/5643703/80639947-c0724900-8a30-11ea-844b-21e2b406df59.png">

Is this still an issue for you? Let me know!

**Comment by JLouSRM** [comment]
Hi @lauralorenz 

Sorry for the late reply!

It is still an issue for me. I used S3 storage, S3ResultHandler, RemoteEnvironment of Dask cluster, and a kubernete agent. Also, I set `@task(log_stdout=True)` to see the result in the logging.

If I hit normal run button either with the default parameters or customized one, I can still see BoxList in the logging. Here is the result:

![logging](https://github.com/JLouSRM/srm_dev/blob/master/boxlist_screenshot.png?raw=True)
